packages/pigeon/pigeons/message.dart

```diff
@@ -16,40 +16,67 @@ import 'package:pigeon/pigeon.dart';
     prefix: 'AC',
   ),
 ))
+
+/// This comment is to test enum documentation comments.
 enum MessageRequestState {
   pending,
   success,
   failure,
 }
 
+/// This comment is to test class documentation comments.
```

Comment:
At least some documentation should be multi-line for manual testing of that case; a class would be a good candidate. So:

```
/// This comment is to test class documentation comments.
///
/// This one has multiple lines.
```